src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -2088,8 +2125,14 @@ public GHLabel createLabel(String name, String color, String description) throws
     }
 
     private PagedIterable<GHUser> listUsers(final String suffix) {
+        Map<String, Object> defaultArgs = Collections.EMPTY_MAP;
+        return listUsers(suffix, defaultArgs);
+    }
+
+    private PagedIterable<GHUser> listUsers(final String suffix, Map<String, Object> args) {
```

Comment:
```suggestion
    private PagedIterable<GHUser> listUsers(final String suffix, GitHubRequest.Builder request) {
```
Instead of passing args, maybe pass the request builder?

Reply:
I'll play around with that, and see if that leads to a more elegant result.

Reply:
I like the `with(...)` method addition.

Reply:
Ok, I think I have a slightly nicer way of passing things around. I think I'll shortly be done addressing feedback, and I'll record for the test and update the PR.